customer_service/index.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+header("Content-Type: application/json");
+header("Access-Control-Allow-Origin: *");
+header("access-control-allow-origin: *");
+header("Access-Control-Allow-Methods: GET,HEAD,OPTIONS,POST,PUT");
+header("access-control-allow-methods: GET,HEAD,OPTIONS,POST,PUT");
+header("Access-Control-Allow-Headers: Or_+_igin, X-Requested-With, Content-Type, Accept, Authorization,Access-Control-Allow-Origin,access-control-allow-headers");
+header("access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization,Access-Control-Allow-Origin,access-control-allow-headers,access-control-allow-methods");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function header() is discouraged](https://app.codacy.com/gh/BuildForSDG/Team-064-Backend/pullRequest?prid=5616370)